File: 백준/Silver/2941. 크로아티아 알파벳/크로아티아 알파벳.java
import java.io.*;

class Main {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		String[] croatia = {"c=", "c-", "dz=", "d-", "lj", "nj", "s=", "z="};
		String word = br.readLine();
		for (String s : croatia) {
			if (word.contains(s)) {
				word = word.replace(s, "A");
			}
		}
		System.out.println(word.length());
    }
}

File: 백준/Silver/2941. 크로아티아 알파벳/크로아티아 알파벳.py
croatia = ['c=', 'c-', 'dz=', 'd-', 'lj', 'nj', 's=', 'z=']
string = input()
count = 0
for i in croatia:
  if string.find(i) > -1:
    string = string.replace(i, "f")
print(len(string))